flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/ManifestOutputFileFactory.java

```diff
@@ -36,22 +36,24 @@
   private final Map<String, String> props;
   private final String flinkJobId;
   private final int subTaskId;
+  private final String operatorUniqueId;
   private final long attemptNumber;
   private final AtomicInteger fileCount = new AtomicInteger(0);
 
   ManifestOutputFileFactory(TableOperations ops, FileIO io, Map<String, String> props,
-                            String flinkJobId, int subTaskId, long attemptNumber) {
+                            String flinkJobId, int subTaskId, String operatorUniqueId, long attemptNumber) {
     this.ops = ops;
     this.io = io;
     this.props = props;
     this.flinkJobId = flinkJobId;
     this.subTaskId = subTaskId;
+    this.operatorUniqueId = operatorUniqueId;
     this.attemptNumber = attemptNumber;
   }
 
   private String generatePath(long checkpointId) {
-    return FileFormat.AVRO.addExtension(String.format("%s-%05d-%d-%d-%05d", flinkJobId, subTaskId,
-        attemptNumber, checkpointId, fileCount.incrementAndGet()));
+    return FileFormat.AVRO.addExtension(String.format("%s-%05d-%s-%d-%d-%05d", flinkJobId, subTaskId,
```

Comment:
nit: might be more consistent if we follow the order of jobId, operatorId, subtaskId

Reply:
+1. Little things like that to be more consistent make developer experience a lot smoother in the aggregate.

Reply:
Agreed, I'll fix it.